documentation/meta/project_boards/index.md

```diff
@@ -0,0 +1,23 @@
+# Project boards
+
+Openverse project management consists of three project boards,
+[one for issues](https://github.com/orgs/WordPress/projects/75),
+[one for PRs](https://github.com/orgs/WordPress/projects/98) and
+[one for project threads](https://github.com/orgs/WordPress/projects/70).
```

Comment:
There's also the "discussions" board, I don't know how that fits into all this though :thinking:

Reply:
Good reminder! I'll at least mention it and document its existing workflows.